nodes.py

```diff
@@ -597,4 +598,18 @@ def upscale(self, image, upscale_method, width, height, crop):
     "CLIPLoader": CLIPLoader,
 }
 
-
+def load_custom_nodes():
+    possible_modules = os.listdir("custom_nodes")
+    possible_modules.remove("put_custom_nodes_here")
```

Comment:
instead of doing that you should only try importing files that end with .py

I would also like it if the put_custom_nodes_here file was a .py.example file instead with a very short example of a custom node.

Reply:
I also want to make it able to load nodes from a folder because in some case, saving all the code into a single file can isn't a good idea, especially when coding complex nodes. But I agree that we should check .py extension.

I agree that it is a good idea to write some example. So, I also wrote two examples of custom nodes. One for importing from file and the other for importing from folder.